iothub_client/samples/iothub_client_sample_module_sender/iothub_client_sample_module_sender.c

```diff
@@ -127,9 +127,14 @@ static void SendConfirmationCallback(IOTHUB_CLIENT_CONFIRMATION_RESULT result, v
 void iothub_client_sample_module_sender(void)
 {
     IOTHUB_CLIENT_LL_HANDLE iotHubClientHandle;
-
     EVENT_INSTANCE messages[MESSAGE_COUNT];
 
+    char *envCS = getenv("EdgeHubConnectionString");
```

Comment:
We tend to not use abbreviations for this, so connectionStringFromEnvironment would be closer.

Reply:
:) I like small variable names but that's me. Changed it to `connectionStringFromEnvironment`.